processing-service-core/api/swagger/swagger.yaml

```diff
@@ -952,7 +954,123 @@ paths:
           description: The specified Workflow could not be found.
           schema:
             $ref: "#/definitions/errorMessage"    
+  ##########################
+  ##  Default Config Store
+  ##########################
+  /globalConfig/{key}:
+    x-swagger-router-controller: globalConfig
+    parameters:
+      - name: key
+        in: path
+        type: string
+        description: The identifier of global config for tenant.
+        required: true
+    put:
+      tags:
+        - Global Config
+      parameters:
+        - name: default
+          type: string
+          in: query
+          description: The value to use when setting the global config for the tenants key.
+          required: true
+        - name: description
+          type: string
+          in: query
+          description: The description of the global configuration value.
+          #required: false
+      summary: Creates global config for the key.
+      description: Creates a global configuration setting for this key or updates the currently existing global configuration setting if it already exists.
+      operationId: createGlobalConfig
+      responses:
+        201:
+          description: Indicates that the global configuration was created successfully.
+        204:
+          description:  Indicates that the global configuration was updated successfully.
```

Comment:
Your controller code is always using 'writeCreatePromiseJSONResultToResponse' which will respond with a 204. Update the contract to only have 204 and change description to mention it was either created or updated.


Anthony is doing this for tenant configs [here](https://github.com/CAFDataProcessing/data-processing-service/blob/SCMOD-3111/processing-service-core/api/swagger/swagger.yaml#L1050).